src/meshes/serial_tree.jl

```diff
@@ -32,6 +32,7 @@ mutable struct SerialTree{NDIMS} <: AbstractTree{NDIMS}
     levels::Vector{Int}
     coordinates::Matrix{Float64}
     original_cell_ids::Vector{Int}
+    mpi_ranks::Vector{Int}
```

Comment:
I don't like introducing MPI-parallel data structures into the plain serial code...

Reply:
That seemed strange to me as well. It was necessary because  `ParallelTree` is used during the simulation whereas SerialTree is used when converting the output via `Trixi2Vtk`. So in order to get the MPI ranks in paraview I had to introduce this field here as well.

In general I find the `mpi_ranks` output very useful. Do you have an idea how to better implement it? Anyways, I will revert these changes here in this PR!

Reply:
You could specialize only the MPI-parallel case without changing the serial infrastructure?